go/client/cmd_team_add_members_bulk.go

```diff
@@ -110,6 +114,14 @@ func (c *CmdTeamAddMembersBulk) ParseArgv(ctx *cli.Context) (err error) {
 		return errors.New("Need at least one of --bots, --readers, --writers, --admins or --owners")
 	}
 	c.arg.SendChatNotification = !ctx.Bool("skip-chat-message")
+
+	emailInviteMsg := ctx.String("email-invite-message")
+	if len(emailInviteMsg) == 0 {
```

Comment:
no need for this branch, just rewrite as 
```
if len(emailInviteMsg) > 0 {
   c.arg.EmailInviteMessage = &emailInviteMsg
}

Reply:
might also want to add similar argument to `go/client/cmd_team_add_member.go` command

Reply:
Cool I'll add to the single `add_member`